addons/mail/static/src/components/follow_button/follow_button.js

```diff
@@ -19,12 +19,7 @@ export class FollowButton extends Component {
              */
             isUnfollowButtonHighlighted: false,
         });
-        useStore(props => {
-            const thread = this.env.models['mail.thread'].get(props.threadLocalId);
-            return {
-                threadIsCurrentPartnerFollowing: thread && thread.isCurrentPartnerFollowing,
-            };
-        });
+useModels();
```

Comment:
misindent

Reply:
I think those are the only 2, tell me if you see more.